GoogleDataTransport/GDTCORLibrary/GDTCOREndpoints.m

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2018 Google
```

Comment:
Should the copyright be 2020?

Reply:
Yeah, it should be `Copyright 2020 Google LLC`, thank you for catching this.

Reply:
Done.